script.js

```diff
@@ -30,39 +29,58 @@
       .attr("width", width)
       .attr("height", height);
 
-    svg.call(tip);
-
-    queue()
-      .defer(d3.json, "../state_squares.geojson")
-      .defer(d3.csv, "/data/map_data.csv", function(d) { rateById.set(d.state, +d.math_diff); })
-      .await(ready);
-
-    // Build map and labels
-    function ready(error, us) {
-      svg.append("g")
-          .attr("class", "states")
-        .selectAll("path")
-          .data(us.features)
-        .enter().append("path")
-          .attr("class", function(d) { return d.properties.abbr; })
-          .style("fill", function(d) { console.log(rateById.get(d.properties.abbr));return color(rateById.get(d.properties.abbr))})
-          .attr("d", path)
-          .on('mouseover', tip.show)
-          .on('mouseout', tip.hide);
-
-      svg.selectAll(".place-label")
-          .data(us.features)
-        .enter().append("text")
-          .attr("class", "place-label")
-          .attr("transform", function(d) { return "translate(" + path.centroid(d) + ")"; })
-          .attr("dy", ".5em")
-          .attr("dx", "-.7em")
-          .text(function(d) { return d.properties.abbr; });
-    }
+    svg.call(tip); */
+
+  queue()
+    .defer(d3.json, "../state_squares.geojson")
```

Comment:
I'm not sure how this is working for you, but this should be `.defer(d3.json, "state_squares.geojson")`.